tools/hooks/python.sh

```diff
@@ -1,15 +1,25 @@
 #!/bin/sh
 # `sh` must be used here instead of `bash` to support GitHub Desktop.
 set -e
+
+# Strip the "App Execution Aliases" from $PATH. Even if the user installed
+# Python using the Windows Store on purpose, these aliases always generate
+# "Permission denied" errors when sh.exe tries to invoke them.
+PATH=$(echo "$PATH" | tr ":" "\n" | grep -v "AppData/Local/Microsoft/WindowsApps" | tr "\n" ":")
+
+# Try to find a Python executable.
 if command -v python3 >/dev/null 2>&1; then
 	PY=python3
 elif command -v python >/dev/null 2>&1; then
 	PY=python
 elif command -v py >/dev/null 2>&1; then
-	PY=py
+	PY="py -3"
 else
-	echo "Please install Python 3.6 or later."
+	echo "Please install Python from https://www.python.org/downloads/"
```

Comment:
Why the removal of the version number?

I had an issue with this where it wasn't working, and this error detailed to me that it was because my python was outdated (I was on 3.5).

Reply:
This is the message for if no Python is found. The message for if a too-old Python is found is still present.